detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/config/validation/AbstractYamlConfigValidator.kt

```diff
@@ -0,0 +1,30 @@
+package io.gitlab.arturbosch.detekt.core.config.validation
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.ConfigValidator
+import io.gitlab.arturbosch.detekt.api.Notification
+import io.gitlab.arturbosch.detekt.core.config.YamlConfig
+
+internal abstract class AbstractYamlConfigValidator(
+    val excludePatterns: Set<Regex> = emptySet(),
+) : ConfigValidator {
```

Comment:
You don't need `excludePatterns` inside `AbstractYamlConfigValidator` any more. You aren't using it.

```suggestion
internal abstract class AbstractYamlConfigValidator : ConfigValidator {
```